### ISSUE TITLE
Not all Math symbols and equations inserted with Insert equation option are recognized.

### ISSUE THREAD

**Issue by poonamdeo** [nvaccess/nvda#11526]
**Summary of the issue:**
In using NVDA with mathematics we have found that there are some symbols not recognized by NVDA.
Math auto-correct for inserting Math symbols in Word.
For example: To get epsilon sign if Math auto-correct enabled typing \epsilon gives ϵ, sign. \perp gives ⊥, etc.
It should read as the symbol name, instead of reading as the respective symbol number from the symbols list.

**System configuration:**
NVDA installed from: https://ci.appveyor.com/api/buildjobs/9kp2iwapkhtlbpx4/artifacts/output%2Fnvda_snapshot_pr11467-20720%2C12322f65.exe
NVDA version: pr11467-20720, 12322f65
TTS synthesizer: eSpeakNG
Windows 10
Word 2016
Add-ons got disabled automatically while installing this pr.

**Description of how this pull request might fix the issue:**
Maybe all symbols available in Math auto-correct list and Word symbols list categories Mathematical operators, Miscellaneous Mathematical Symbols–A, Miscellaneous Mathematical Symbols–B and Supplemental Mathematical Operators, and arrows are there in the symbols list as these arrows get used in writing Chemical expressions in Word if added to the NVDA dictionary it would help the user access most used format "Word" for STEM subjects without needing to rely on any of the other software.

The below-mentioned issue is linked to issue number: #4631

NVDA doesn't read Math expressions inserted in Word insert equation editor.
Even the symbols recognized so far are when inserted in equation editor NVDA do not recognize the equation or characters in it.
It announces as a character number.

**Comment by DrSooom** [comment]
@poonamdeo: Please completely fill out the bug report draft and add there your used language (OS, Microsoft Office NVDA, TTS synthesizer) and the used TTS synthesizer (incl. version) as well. Thanks.

**Comment by lukaszgo1** [comment]
the second part of this issue i.e. symbols inserted by the equation editor not being readable with NVDA is a duplicate of #4631

**Comment by poonamdeo** [comment]
Hi&nbsp;Daniel,I didn&#39;t fully understand your message, what I understood is I will need to add the system specifications nothing else right?Thank you!Poonam DeokarConsulting Special EducatorXavier&#39;s Resource Centre&nbsp;for the Visually Challenged&nbsp;(XRCVC)St.Xavier&#39;s College5, Mahapalika MargMumbai 400001# 022-22623298/22623242From: Daniel Mayr &lt;notifications@github.com&gt;Sent: Sat, 29 Aug 2020 21:04:34 GMT+0530To: nvaccess/nvda &lt;nvda@noreply.github.com&gt;Cc: Poonam Deokar &lt;poonam@xrcvc.org&gt;, Mention &lt;mention@noreply.github.com&gt;Subject: Re: [nvaccess/nvda] Not all Math symbols and equations inserted with Insert equation option are recognized. (#11526)&nbsp;&nbsp;
&nbsp;@poonamdeo: Please completely fill out the bug report draft and add there your used language (OS, Microsoft Office NVDA, TTS synthesizer) and the used TTS synthesizer (incl. version) as well. Thanks.
&mdash;You are receiving this because you were mentioned.Reply to this email directly,view it on GitHub, orunsubscribe.s

**Comment by poonamdeo** [comment]
Hi Lukasz,Thank you for the information, will go through this ticket thread and see what works for us.Regards,Poonam DeokarConsulting Special EducatorXavier&#39;s Resource Centre&nbsp;for the Visually Challenged&nbsp;(XRCVC)St.Xavier&#39;s College5, Mahapalika MargMumbai 400001# 022-22623298/22623242From: Łukasz Golonka &lt;notifications@github.com&gt;Sent: Sat, 29 Aug 2020 21:27:29 GMT+0530To: nvaccess/nvda &lt;nvda@noreply.github.com&gt;Cc: Poonam Deokar &lt;poonam@xrcvc.org&gt;, Mention &lt;mention@noreply.github.com&gt;Subject: Re: [nvaccess/nvda] Not all Math symbols and equations inserted with Insert equation option are recognized. (#11526)&nbsp;&nbsp;
the second part of this issue i.e. symbols inserted by the equation editor not being readable with NVDA is a duplicate of#4631
&mdash;You are receiving this because you were mentioned.Reply to this email directly,view it on GitHub, orunsubscribe.s

**Comment by Adriani90** [comment]
I think this will also improve significantly after #11467 has been merged.

**Comment by DrSooom** [comment]
@poonamdeo: It is always recommended to completely fill out the [bug report draft](https://github.com/nvaccess/nvda/issues/new?template=bug_report.md) for bugs in NVDA. This template also includes headlines, which improves the clarity of an issue. And it makes it easier for developers to reproduce the issue.

**Comment by Adriani90** [comment]
@poonamdeo this is now much better after unicode normalization is enabled in NVDA voice settings. Closing as works for me.
If you are still having this issue or you find symbols that are not recognized, please comment with a list of symbols that you would like to have spoken and we can reopen.
